File: SenceAndroid/M4_Clase6_1/app/src/main/java/com/sudoagile/m4_clase6_1/MainActivity.java
package com.sudoagile.m4_clase6_1;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import java.util.ArrayList;
import java.util.List;



public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        init();
    }

    public void init() {
        element = new ArrayList<>();
        element.add(new ListElement(
                color: "#06141B",
                name: "Clase 5",
                subtitle: "RecyclerView",
                status: "Activo")
        );

        ListAdapter listAdapter = new ListAdapter(element, this);
        RecyclerView recyclerView = findViewById(R.id.rvList);
    }
}

File: SenceAndroid/ZoologicoApp/app/src/main/java/com/codigo/zoologicoapp/MainActivity.kt
package com.codigo.zoologicoapp

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Scaffold
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.codigo.zoologicoapp.ui.theme.ZoologicoAppTheme

import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main) // ✅ Mantener XML
    }
}


   // override fun onCreate(savedInstanceState: Bundle?) {
     //   super.onCreate(savedInstanceState)
      //  enableEdgeToEdge()
      //  setContent {
        //    ZoologicoAppTheme {
          //      Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
            //        Greeting(
              //          name = "Android",
                //        modifier = Modifier.padding(innerPadding)
                  //  )
             //   }
         //   }
      //  }
  //  }
//}

//@Composable
//fun Greeting(name: String, modifier: Modifier = Modifier) {
 //   Text(
  //      text = "Hello $name!",
   //     modifier = modifier
  //  )
//}

//@Preview(showBackground = true)
//@Composable
//fun GreetingPreview() {
  //  ZoologicoAppTheme {
    //    Greeting("Android")
   // }
